secantTests/MultiLineTextFieldTests/MultiLineTextFieldTests.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  MultiLineTextFieldTests.swift
+//  secantTests
+//
+//  Created by Lukáš Korba on 01.08.2022.
+//
+
+import XCTest
+@testable import secant_testnet
+import ComposableArchitecture
+
+class MultiLineTextFieldTests: XCTestCase {
+    func testIsCharLimitted() throws {
+        let store = TestStore(
+            initialState: MultiLineTextFieldState(charLimit: 1),
+            reducer: MultiLineTextFieldReducer.default,
+            environment: MultiLineTextFieldEnvironment()
+        )
+        
+        store.send(.binding(.set(\.$text, "test"))) { state in
+            state.text = "test"
+            XCTAssertTrue(
+                state.isCharLimitted,
+                "Multiline TextFiler tests: `testIsCharLimitted` is expected to be true but it is \(state.isCharLimitted)"
+            )
+        }
+    }
+
+    func testIsNotCharLimitted() throws {
```

Comment:
```suggestion
    func testIsNotCharLimited() throws {
```

Reply:
there were actually other instances of 2 TT, I fixed all instances